graviti/schema/package.py

```diff
@@ -141,6 +154,21 @@ def __init__(self, package: ExternalPackage) -> None:
         self.package = package
         self._data: Dict[str, Type["PortexExternalType"]] = {}
 
+    def __setitem__(self, key: str, value: Type["PortexExternalType"]) -> None:
+        type_ = self._data.get(key)
+        if type_:
+            if type_ != value:
+                raise KeyError("Duplicate names")
+            return
+
+        for name, type_ in self._data.items():
+            if type_ == value:
+                if name != key:
+                    raise ValueError("Same type with the different names in subpackage")
```

Comment:
Would "Same type with a different name already exists in the subpackage" be more clear?